internal/workflow_testsuite.go

```diff
@@ -65,8 +65,9 @@ type (
 		call *mock.Call
 		env  *TestWorkflowEnvironment
 
-		runFn        func(args mock.Arguments)
-		waitDuration time.Duration
+		runFn          func(args mock.Arguments)
+		waitDuration   time.Duration
+		waitDurationFn func() time.Duration
```

Comment:
A possible code simplification is to store only the Fn and make After set the Fn that returns a passed value.

Reply:
Yeah, I'll do that.  Other stuff has func-wrappers in here, and it'd be good to get rid of some branches.